Description: Loosen env_logger version requirement
Author: Abhinaba Rakshit <abhinaba.rakshit@oss.qualcomm.com>
Origin: other, Debian packaging
Last-Update: 2026-04-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: mink-idl-compiler/idlc_errors/Cargo.toml
===================================================================
--- mink-idl-compiler.orig/idlc_errors/Cargo.toml
+++ mink-idl-compiler/idlc_errors/Cargo.toml
@@ -6,5 +6,5 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-env_logger = "0.11.3"
+env_logger = ">= 0.10.0, < 0.12.0"
 log = "0.4.21"
